localisation/00_zuwc_united_localisation.csv

```diff
@@ -2848,10 +2848,10 @@ GetMonstrousUndeadNameDesc_Abomination;a horrific creation that is sewn together
 GetMonstrousUndeadNameDesc_WickerGolem;a hulking creation that is crafted from materials such as flesh, fur and bone and animated through an ancient death magic;;eine grobe Sch�pfung, die aus toten Materialien wie Fleisch, Fell und Knochen hergestellt und durch eine alte Todesmagie belebt wird;;;;;;;;;;;x
 EVTDESC_WCPOD_5013_wickerman;Your days in a living body come to an end. Thros offered  you a new life, giving you the opportunity to tie your soul to the wicker revenant, removing the limitations and weaknesses of flesh.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros hat Euch ein neues Leben geschenkt und Euch die M�glichkeit gegeben, Eure Seele an den Weidenwiederg�nger zu binden und die Begrenzungen und Schw�chen des Fleisches zu �berwinden.;;;;;;;;;;;x
 EVTDESC_WCPOD_5013_wicker_golem;Your days in a living body come to an end. Thros offered you a new life, giving you the opportunity to tie your soul to the wicker golem, greatly improving your physical strength and turning you into a fearsome machine of war.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros schenkte Euch ein neues Leben und gab Euch die M�glichkeit, Eure Seele an den Weidengolem zu binden, was Eure K�rperkraft stark erh�hte und Euch in eine furchteinfl��ende Kriegsmaschine verwandelte.;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTTITLE_WCORD_74;Creation of Blood Knights Order;;;;;;;;;;;;;x
+EVTDESC_WCORD_74;After many centuries of fighting against trolls and others high elfs established and craved from bones of their enemies new Empire for all world to envy. Quel'Thalas, eversummer land, now is threatened by new threats. Trolls seeking vengeance for their bretheren, vile undead trying to complete their master's plan, evil demons trying to destroy Azeroth and awakened Old Gods. All trying to destroy new home of elfs, even xenophobic humans want to see Quel'Thalas burn. No longer able to wait [From.GetFirstName] created Blood Knights Order, Order of elven paladins that will help defend Quel'Thalas and Sunwell against any outside threat. All around realm paladins bring their weapons and faith to Silvermoon city and join Blood Knights ranks. Light shines above Silvermoon city.;;;;;;;;;;;;;x
+nick_the_sunblade;the Sunblade;;;;;;;;;;;;;x
+EVTOPTA_WCORD_74;May light be with them;;;;;;;;;;;;;x
```

Comment:
Did you write this line on the 4th of May? :D

Reply:
No, this was kinda random lol